app/views/comments/index.html.erb

```diff
@@ -1,23 +1,9 @@
 <!--Descriptive page name, messages and instructions-->
-<% if @commentable.nil? %>
-  <!-- the most recent comments -->
-  <% cache(logged_in? ? 'latest_comments' : 'latest_comments_public', skip_digest: true) do %>
```

Comment:
for the cache - I wasn't sure if removing `latest_comments`/`latest_comments_public` was something we could do here as well. 

The only place I see it being used is in https://github.com/otwcode/otwarchive/blob/master/app/sweepers/comment_sweeper.rb#L13 but I wasn't sure how that was being used! (wasn't sure if that was being used to show comments on individual works or if that would be safe to be removed here/a related PR?)

Reply:
ouu good point! i'll give removing it a go :D